yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -12,6 +12,9 @@
 
 <div class = box>
     <link rel="stylesheet" type="text/css" href="{% static 'design/detailed_design.css' %}">
+    <div class = workflow_id>
```

Comment:
i'm not going to request you change this for this pull request, but in the future for consistencies sake, the generally accepted style for html files is `class=workflow_id` instead of `class = workflow_id`.